public/app/features/plugins/extensions/getPluginExtensions.test.tsx

```diff
@@ -564,3 +575,174 @@ describe('getPluginExtensions()', () => {
     );
   });
 });
+
+describe('getObservablePluginExtensions()', () => {
+  const extensionPointId = 'grafana/dashboard/panel/menu/v1';
+  const pluginId = 'grafana-basic-app';
+
+  beforeEach(() => {
+    pluginExtensionRegistries.addedLinksRegistry = new AddedLinksRegistry();
+    pluginExtensionRegistries.addedComponentsRegistry = new AddedComponentsRegistry();
+    pluginExtensionRegistries.addedLinksRegistry.register({
+      pluginId,
+      configs: [
+        {
+          title: 'Link 1',
+          description: 'Link 1 description',
+          path: `/a/${pluginId}/declare-incident`,
+          targets: extensionPointId,
+          configure: jest.fn().mockReturnValue({}),
+        },
+      ],
+    });
+
+    pluginExtensionRegistries.addedComponentsRegistry.register({
+      pluginId,
+      configs: [
+        {
+          title: 'Component 1',
+          description: 'Component 1 description',
+          targets: extensionPointId,
+          component: () => {
+            return <div>Hello world!</div>;
+          },
+        },
+      ],
+    });
+  });
+
+  it('should emit the initial state when no changes are made to the registries', () => {
+    const observable = getObservablePluginExtensions({ extensionPointId });
+
+    observable.subscribe((value: ReturnType<GetExtensions>) => {
+      expect(value.extensions).toHaveLength(2);
+      expect(value.extensions[0].pluginId).toBe(pluginId);
+      expect(value.extensions[1].pluginId).toBe(pluginId);
+    });
```

Comment:
This is usually not an issue and this is probably fine, but there are some gotchas with testing observables that the matchers [`toEmitValues`](https://github.com/grafana/grafana/blob/main/public/test/matchers/toEmitValues.ts) or [`toEmitValuesWith`](https://github.com/grafana/grafana/blob/main/public/test/matchers/toEmitValuesWith.ts) take care of more safely. Maybe you should try them out?